pkg/deployment/inventory.go

```diff
@@ -250,3 +271,55 @@ func buildNetworkVars(networks []infranetworkv1.IPSetNetwork) ([]string, map[str
 	}
 	return nets, netsLower
 }
+
+// check if string is a valid ansible variable name
+func isAnsibleVar(varname validator.FieldLevel) bool {
+	// Python keywords can not be used as variable names
+	// in future this list may be subject to ammendement, pruning, or other change
+	switch varname.Field().String() {
+	case
+		"False",
+		"await",
+		"else",
+		"import",
+		"pass",
+		"None",
+		"break",
+		"except",
+		"in",
+		"raise",
+		"True",
+		"class",
+		"finally",
+		"is",
+		"return",
+		"and",
+		"continue",
+		"for",
+		"lambda",
+		"try",
+		"as",
+		"def",
+		"from",
+		"nonlocal",
+		"while",
+		"assert",
+		"del",
+		"global",
+		"not",
+		"with",
+		"async",
+		"elif",
+		"if",
+		"or",
+		"yield":
+		return false
```

Comment:
I really don't think we need to be keeping a list of python keywords in dataplane-operator. ansible variable validation is for ansible to do...not dataplane-operator. I understand the utility in trying to validate everything as early as possible, but not at the cost of having to write and maintain validation logic that is specific to an entirely separate tool (ansible in this case).